cse/SearchEngine.py

```diff
@@ -140,9 +141,13 @@ def __booleanSearch(self, query):
         # load document set per term
         for term in terms:
             cidTuples = []
-            if term.endswith("*"):
+            if term.strip().endswith("*"):
                 cidTuples = self.__prefixSearchTerm(term.replace("*", ""))
 
+            elif term.strip().startswith("ReplyTo:"):
```

Comment:
Do we have a replyto search within boolean queries?

Reply:
> As a bonus task, you could also answer queries of the type: `ReplyTo:12345678 AND fake`,
> which returns the comments that reply to the comment with id `12345678` and contain the
> word `fake`.

line [148](https://github.com/CodeLionX/CommentSearchEngine/pull/60/files#diff-d8ccb5fcd9260c166fc64e9c1a9773cbR148) implements this...
An example query can be found in line [386](https://github.com/CodeLionX/CommentSearchEngine/pull/60/files#diff-d8ccb5fcd9260c166fc64e9c1a9773cbR386)